components/invitation_modal/invitation_modal.tsx

```diff
@@ -161,10 +161,14 @@ export class InvitationModal extends React.PureComponent<Props, State> {
     invite = async () => {
         const roleForTrackFlow = getRoleForTrackFlow();
         const inviteAs = this.state.invite.inviteType;
+        const eventProps = {
+            num_invitations: this.state.invite.usersEmails.length, 
+            ...roleForTrackFlow
+        };
         if (inviteAs === InviteType.MEMBER && this.props.isCloud) {
-            trackEvent('cloud_invite_users', 'click_send_invitations', {num_invitations: this.state.invite.usersEmails.length, ...roleForTrackFlow});
+            trackEvent('cloud_invite_users', 'click_send_invitations', eventProps);
```

Comment:
@ifoukarakis Looks like the cloud team was running a test here. Not sure whether to keep or remove? The telemetry below should cover it because we have the `inviteType` as a prop

cc/ @marianunez

Reply:
@BenCookie95 @ifoukarakis We currently don't have any A/B tests running for user invites. This may just be that at some point we were tracking specific to cloud but I don't think this is needed given that this can be determined by seeing the server where this is coming from. Feel free to modify as needed. 

For reference: https://docs.google.com/spreadsheets/d/17MpGsrrfbBPHT_GW12lTqI5J24XGRBMTerwKZcZ1zDI/edit?usp=sharing

Reply:
Thanks Maria, I'm going to remove this particular event